roadmaps/2018.md

```diff
@@ -1,7 +1,19 @@
 # Majifix Roadmap 2018
 
+This page outlines work that is currently being worked on and things we plan to work on next, within this year of 2018. That said, we do value feedback from the community to help us better understand user needs and adjust priorities.
+
 ## Majifix API
 
+Majifix API is built on Node.JS. It is a RESTful API layer designed in support of reporting and tracking common public-service (i.e. water) problems.
```

Comment:
For this section, It is also nicely done but we should move it up to the introduction part where we explain about majifix components. We can expand components descriptions to be paragraphs in-stead of being just a single or two lines.

Reply:
@lykmapipo If you have time please create a skeleton for the roadmap for the API so that the structure can be easily replicated to other majifix products

Reply:
@BenMaruchu I appreciate the feedback.

- I think moving it to the introduction part is wise.
- I think road-map will make much more sense, to show what is to be shipped, even if itemized (no paragraphs). @lykmapipo Maybe the skeleton should consider this.

Reply:
@joshuamabina & @BenMaruchu Sure. Roadmap template will make sense. Please @joshuamabina initiate and I will guide.